test/index.test.js

```diff
@@ -669,6 +669,122 @@ Previous tag: ''
 
         expect.assertions(1)
       })
+
+      it('categorizes pull requests with overlapping labels', async () => {
+        getConfigMock('config-with-categories-3.yml')
+
+        nock('https://api.github.com')
+          .get('/repos/toolmantim/release-drafter-test-project/releases')
+          .query(true)
+          .reply(200, [require('./fixtures/release')])
+
+        nock('https://api.github.com')
+          .post('/graphql', body =>
+            body.query.includes('query findCommitsWithAssociatedPullRequests')
+          )
+          .reply(
+            200,
+            require('./fixtures/__generated__/graphql-commits-overlapping-label.json')
+          )
+
+        nock('https://api.github.com')
+          .post(
+            '/repos/toolmantim/release-drafter-test-project/releases',
+            body => {
+              expect(body).toMatchInlineSnapshot(`
```

Comment:
We should use `toMatchInlineSnapshot` for all our other inline matching 😅

Reply:
That's a great idea!